plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestingThriftHiveMetastoreBuilder.java

```diff
@@ -62,7 +72,8 @@ public TestingThriftHiveMetastoreBuilder metastoreClient(HostAndPort address)
     {
         requireNonNull(address, "address is null");
         checkState(tokenAwareMetastoreClientFactory == null, "Metastore client already set");
-        tokenAwareMetastoreClientFactory = new TestingTokenAwareMetastoreClientFactory(HiveTestUtils.SOCKS_PROXY, address);
+        tokenAwareMetastoreClientFactory = new TestingTokenAwareMetastoreClientFactory(
+                HiveTestUtils.SOCKS_PROXY, URI.create("thrift://" + address.getHost() + ":" + address.getPort()));
```

Comment:
`metastoreClient(URI metastoreUri)` can be used instead of code repetition

Reply:
done